### PR TITLE
Deprecated playbooks and scripts description improvement to correlate the new validation 

### PR DESCRIPTION
**Pull request by ShacharKidor**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/37315
## Description
- Descriptions of deprecated playbooks and scripts were edited in order to correlate a new validation in Demisto-SDK validate.
- Playbooks and scripts of Deprecated content pack which relevant to the new validation were added to .pack-ignore file.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/DeprecatedContent/Scripts/script-VectraTriage.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ type: python
 subtype: python2
 tags:
 - Vectra
-comment: Deprecated. The rules branch can be used to retrieve a listing of configured Triage rules
+comment: The rules branch can be used to retrieve a listing of configured Triage rules
```
Comment (Itay4) [inline Packs/DeprecatedContent/Scripts/script-VectraTriage.yml:24]: why did you remove the deprecated note here?

**Review by Itay4 — COMMENTED** [review]
1. please change all release notes to `Documentation and metadata improvements.`
2. please make sure the `No available replacement.` is after the `Deprecated.` and before the description itself

File: Packs/DeprecatedContent/Scripts/script-VectraTriage.yml (hunk 1 above)
Comment (ShacharKidor) [reply Packs/DeprecatedContent/Scripts/script-VectraTriage.yml]: By mistake. Fixed.

**Review by Itay4 — APPROVED** [review]
for the sane report scripts, no need to mention in the description they're deprecated so lets ignore the validation on them

**Comment by Itay4** [comment]
build fails due to https://github.com/demisto/etc/issues/26905
merging
